header-verifier/tests/header_verifier_blackbox_tests.rs

```diff
@@ -821,15 +826,18 @@ fn test_change_validator_invalid_epoch() {
     let validator_set = MultiValueEncoded::new();
     let epoch = 0u64;
 
+    let expected_logs = Some(vec![
+        log!(CHANGE_VALIDATOR_SET_ENDPOINT, topics: [EXECUTED_BRIDGE_OP_EVENT], data: [INVALID_EPOCH]),
+    ]);
+
     state.change_validator_set(
         &signature,
         &hash_of_hashes,
         &operation_hash,
         epoch,
         &bitmap,
         validator_set,
-        Some(EXECUTED_BRIDGE_OP_EVENT),
-        Some(INVALID_EPOCH),
+        expected_logs,
```

Comment:
To no defined the same `expected_log` variable here...
For this particular function, I think we can just send `execution_error: &str => Some(ERROR) or None` as argument and create the `expected_logs` inside which contains:
- identifier: CHANGE_VALIDATOR_SET_ENDPOINT
- topics: EXECUTED_BRIDGE_OP_EVENT, hash_of_hashes, operation_hash
- data: the error argument

Reply:
This might also work for other simple operations.